base-centos-6.9/main.tf

```diff
@@ -0,0 +1,28 @@
+
+# variable for the ami I want to create
+# let the user enter it
+
+# ami-0a381e9274ee9f1e5
+
+variable "base_ami" { }
```

Comment:
Because we don't specify a base_ami the user will be asked for one when you run `terraform apply` and `terraform destroy`. This value comes from the result of the packer script.

If this were to be automated a little more work would need to be done.